views/blogs/index.ejs

```diff
@@ -28,5 +28,8 @@
         <% }); %>
     </div>
 </div>
+<footer>
 
-<% include ../partials/footer %>
\ No newline at end of file
+    <% include ../partials/footer %>
+
+</footer>
```

Comment:
`footer` tag in each page is redundant.

Reply:
@thebishaldeb  I have Removed now

Reply:
Check the changes I suugested in the #41 issue. Also pleae open one PR at a time, because everything seems messed up now.